packages/vite/src/node/index.ts

```diff
@@ -5,7 +5,7 @@ export { optimizeDeps } from './optimizer'
 export { send } from './server/send'
 export { createLogger } from './logger'
 export { resolvePackageData, resolvePackageEntry } from './plugins/resolve'
-export { normalizePath } from './utils'
+export * from './utils'
```

Comment:
I don't think that it will work that way, this also could lead to naming collisions

Maybe you can use `import { ... } from 'vite/node/utils'` for now? (Haven't tested that)  
But this will not be officially supported and if there are breaking changes, there is currently no support for that

Reply:
It's pre-bundled 😞